shlide

```diff
@@ -75,7 +75,10 @@ display() {
     printf '\e[2J'
 
     # Get screen size.
-    read -r LINES COLUMNS < <(stty -F /dev/tty size)
+    case "$(uname -s)" in
+        Darwin) read -r LINES COLUMNS < <(stty -f /dev/tty size) ;;
```

Comment:
bash has a builtin environment variable `OSTYPE`, we can use this instead of shelling out to `uname`. Here's an example: https://github.com/dylanaraps/pure-bash-bible#simpler-case-statement-to-set-variable

Reply:
Done